core/src/torii/mod.rs

```diff
@@ -347,6 +368,33 @@ async fn handle_get_configuration<W: WorldTrait>(
     .map_err(Error::Config)
 }
 
+/// Message acceptable for `POST` requests to the configuration endpoint.
+#[derive(Clone, Debug, Deserialize, Serialize, Copy)]
+pub enum PostConfiguration {
+    /// Change the maximum logging level of logger.
+    ///
+    /// # Examples
+    /// `curl -X POST -H 'content-type: application/json' http://127.0.0.1:8080/configuration -d '{"ChangeLogLevel": "DEBUG"}' -i`
```

Comment:
This fails for now though. Am I missing something?

![2021-11-17T183629+090000](https://user-images.githubusercontent.com/49983831/142176352-1bc7f42a-17f7-44f1-8cec-9e3a7e85c2e3.png)

Reply:
This is indeed a bug, but I can't seem to reproduce it reliably.